README.md

```diff
@@ -89,6 +89,21 @@ with body:
 
 You'll receive a `200 OK` with a new `access_token` and `refresh_token` in the response body.
 
+### Single Sign-On (SSO)
+
+Using the access token from the user you would like to login as, you can use SSO in two ways. You can either send the `access_token` in the `Authorization` header:
+
+```
+(POST|GET) /sso
+Authorization: Bearer ACCESS_TOKEN
+```
+
+or use the `access_token` parameter:
+
+```
+(POST|GET) /sso?access_token=ACCESS_TOKEN
```

Comment:
For a POST request, the parameter would be in the request body.

Reply:
Good point. I'll split that out.